src/modules/src/Eryph.Modules.ComputeApi/Model/OperationSpecBuilder.cs

```diff
@@ -28,10 +28,12 @@ public ISingleResultSpecification<OperationModel> GetSingleEntitySpec(OperationR
 
```

Comment:
Is it intended that `GetSingleEntitySpec` does not use the variant of `OperationSpecs.GetById` which performs a permission check?

Reply:
nope, good point. Changed and added tests